src/Pos/Wallet/Web/Tracking.hs

```diff
@@ -128,7 +129,7 @@ syncWalletsAtStartWebWallet :: WalletTrackingEnv ext m => [EncryptedSecretKey] -
 syncWalletsAtStartWebWallet = syncWalletsWithGStateLock @WalletSscType
 
 syncOnImportWebWallet :: WalletTrackingEnv ext m => EncryptedSecretKey -> m ()
-syncOnImportWebWallet = selectAccountsFromUtxoLock @WalletSscType . one
+syncOnImportWebWallet = (() <$) . selectAccountsFromUtxoLock @WalletSscType . one
```

Comment:
I rebased a change into a commit (fixing CI by removing unused imports), and now some other commits show up with my name under them. This has happened once before, I'll need to look into it to fix the history.